test/StakeManager.js

```diff
@@ -1501,5 +1484,45 @@ describe('StakeManager', function () {
       const tx = stakeManager.connect(signers[10]).delegate(stakerIdAcc4, amount);
       await assertRevert(tx, 'Staker is slashed');
     });
+
+    it('Staker should be able to stake amount less than minStake', async function () {
```

Comment:
hey now imo mistake is not deciding parameter here 
it should be checked for ><= minSafeRazor

Reply:
@abhishekvispute ok will check for that too.